nailgun/entities.py

```diff
@@ -4298,11 +4298,17 @@ def path(self, which=None):
             /api/hosts/:host_id/module_streams
         disassociate
             /api/hosts/:host_id/disassociate
+        assign_ansible_roles
+            /api/hosts/:host_id/assign_ansible_roles
+        ansible_roles
+            /api/hosts/:host_id/ansible_roles
```

Comment:
Could you extend similar endpoints for hostgroups `/api/hostgroups/:id/ansible_roles` and `/api/hostgroups/:id/assign_ansible_roles`?

Reply:
I can also do this. As above, this was meant to be an MVP to get some testing going elsewhere